### ISSUE TITLE
Calling `brew tap --full` is disabled! There is no replacement.

### ISSUE THREAD

**Issue by bruuuuuuuce** [Homebrew/homebrew-test-bot#688]
I have been trying to build bottles for my homebrew tap. I am using the github actions created by the `brew tap-new` command. There seems to be an issue with the default workflows that get created using `brew tap-new`.

When I run, `brew tap -h`, I see that `--full` is a valid flag to pass to that command, but when trying to use it `brew tap aws/tap --full`, I get the error
```
Calling `brew tap --full` is disabled! There is no replacement.
```

In doing some research, it appears the `brew tap USER/TAP` already does a full clone by default, and only does a shallow clone if you pass in the `--shallow` flag. 

This issue is resulting in my pr builds to fail because under the hood `brew test-bot` calls `brew tap USER/TAP --full`.

https://github.com/Homebrew/homebrew-test-bot/blob/e86c0e65fcb51edb5c85247fe361e3d166b8a7f6/lib/test_bot.rb#L79

I think this issue can be solved just by removing `--full` from the above line, but I don't have enough homebrew knowledge to be sure of that.

**Comment by carlocab** [comment]
That sounds right. Can you open a pull request?

**Comment by bruuuuuuuce** [comment]
yea sure

**Comment by carlocab** [comment]
Closed in #689.
